recipes/bigdft/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "bigdft-suite" %}
+{% set version = "1.9.3_rc.2" %}
+{% set build = 0 %}
+{% set mpi = mpi %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/l_sim/bigdft-suite/-/archive/{{ version }}/bigdft-suite-{{ version }}.tar.gz
+  md5: a756c1673c45e7e7b7be3cdf6aef81b1
+  patches:
+    - patches/output.patch
+    - patches/utils.patch
+
+build:
+  number: {{ build }}
+  skip: true  # [win or osx]
```

```suggestion
build:
  number: {{ build }}
  skip: true  # [win or osx]
  # add build string so packages can depend on
  # mpi or nompi variants explicitly:
  # `pkg * mpi_mpich_*` for mpich
  # `pkg * mpi_*` for any mpi
  # `pkg * nompi_*` for no mpi
  {% if mpi != 'nompi' %}
  {% set mpi_prefix = "mpi_" + mpi %}
  {% else %}
  {% set mpi_prefix = "nompi" %}
  {% endif %}
  string: "{{ mpi_prefix }}_py{{ py }}h{{ PKG_HASH }}_{{ build }}"
```
Please indicate the MPI variant in the build string

https://conda-forge.org/docs/maintainer/knowledge_base.html#preferring-a-provider-usually-nompi

---

There is no `nompi` version of our package, it must be installed with mpi. So in our case this seems unnecessary. Do you recommend instead just putting `{% set mpi_prefix = "mpi_" + mpi %}`? Or to just leave it all out?

---

This is just a suggestion from the docs page. Please modify to fit your use case. You have separate variants for `mpich` and `openmpi` and they should be noted in the build string.

---

Thanks. I've made the modification accordingly, and will continue to work on the other suggestions.